scripts/webpack.config.js

```diff
@@ -3,6 +3,7 @@ const path = require('path');
 const CircularDependencyPlugin = require('circular-dependency-plugin');
 const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
```

Comment:
removing storybook dependency removes node.js modules `util` and `url` which are used by hapi/address